example-app/src/main/java/backtraceio/backtraceio/MainActivity.java

```diff
@@ -99,6 +100,9 @@ private BacktraceClient initializeBacktrace(final String submissionUrl) {
         BacktraceExceptionHandler.enable(backtraceClient);
         // backtraceClient.send("test");
 
+        //(RequestHandler customerHandler = new CustomRequestHandler(credentials);
```

Comment:
can we remove dead code?

Reply:
It isn't dead code, it's like other places where we have commented out code to use as an example.

Reply:
It's dead code please delete it